GEOSldas_App/ldas_setup

```diff
@@ -250,21 +251,33 @@ class LDASsetup:
                exit("Error. LANDASSIM_DT is inconsistent with ADAS analysis window.\n")
            self.rqdExeInp['LANDASSIM_T0']             = "013000"                                 # HHMMSS
            jsgmt1                                     = "00000000"
-           jsgmt2                                     = hours_to_hhmmss(int(self.varwindow)/60)       # convert minutes to HHMMSS
+           jsgmt2                                     = hours_to_hhmmss(int(self.varwindow)/60)  # convert minutes to HHMMSS
            self.rqdExeInp['JOB_SGMT']                 = f"{jsgmt1} {jsgmt2}"
            self.rqdExeInp['NUM_SGMT']                 = 1 
            self.rqdExeInp['FORCE_DTSTEP']             = 3600
 
            # determine END_DATE = BEG_DATE + TIME_STEP_OF_ADAS_CYCLE
-           _beg_date = datetime.strptime( self.rqdExeInp['BEG_DATE'], "%Y%m%d %H%M%S")
-           _hours    = int(self.rqdExeInp['JOB_SGMT'][ 9:11])
-           _end_date = _beg_date + timedelta(hours=int(self.varwindow)/60) 
-
+           _beg_date                                  = datetime.strptime( self.rqdExeInp['BEG_DATE'], "%Y%m%d %H%M%S")
+           _hours                                     = int(self.rqdExeInp['JOB_SGMT'][ 9:11])
+           _end_date                                  = _beg_date + timedelta(hours=int(self.varwindow)/60) 
            self.rqdExeInp['END_DATE']                 = _end_date.strftime("%Y%m%d %H%M%S") 
-           self.rqdExeInp['NML_INPUT_PATH']           = self.LDAS_nml_path
-           self.rqdExeInp['MWRTM_PATH']               = self.mwrtm_path   
+
+           # process LDAS special nml input files
+           # FOR NOW, DEFINE/CREATE etc AND run DIRECTORIES HERE; NOTE THAT self.rundir IS DEFINED
+           # IN __init__() BELOW AND etcdir IS DEFINED IN createRCFiles().
```

Comment:
My implementation is ugly in that I'm defining _rundir and _etcdir here, but both are also defined elsewhere.   Needs cleanup.